shared-bindings/watchdog/WatchDogTimer.c

```diff
@@ -108,41 +110,26 @@ MP_PROPERTY_GETSET(watchdog_watchdogtimer_timeout_obj,
 //|     mode: WatchDogMode
 //|     """The current operating mode of the WatchDogTimer `watchdog.WatchDogMode`.
 //|
-//|     Setting a WatchDogMode activates the WatchDog::
+//|     Setting a `WatchDogMode` activates the WatchDog::
 //|
-//|       import microcontroller
-//|       import watchdog
+//|       from microcontroller import watchdog as w
```

Comment:
we don't usually use this style of "abbreviate via import as" in docs. do you mind writing it "watchdog" in full in this example?